src/FixedDataTableCellGroup.js

```diff
@@ -73,21 +131,34 @@ class FixedDataTableCellGroupImpl extends React.Component {
     this._initialRender = false;
   }
 
+  getColumnGroupWidth = () => {
```

Comment:
Right now it's a bit confusing that getColumnGroupWidth returns a key array as well as a width array. We might want to revisit this later. 
Please add a NOTE here so we can discuss the API later